backend/db.js

```diff
@@ -39,6 +41,11 @@ const Todos = {
 
   toggleComplete: (id) => {
     return db.none(TOGGLE_COMPLETE, [id])
+  },
+  // lowTodoParams = [lowTodoId, higherTodoPirority]
+  // highTodoParams = [highTodoId, lowTodoPirority]
+  swapPriorities: ( lowTodoParams, highTodoParams ) => {
```

Comment:
I would explicitly pass in 4 arguments here and name them `lowTodoId, lowTodoPriority, highTodoId, highTodoPriority`. Naming the params`lowTodoParams, highTodoParams` makes them ambiguous, and makes me think about what they could be. Also if you explicitly name the parameters, the comment becomes redundant.